mex/backend/graph/cypher/fetch_merged_items.cql

```diff
@@ -0,0 +1,69 @@
+<# Fetch merged items including their nested objects and referenced identifiers.
+
+Globals:
+    extracted_labels: List of all extracted class labels
+    rule_labels: List of all rule class labels
+    merged_labels: List of all merged class labels
+    nested_labels: List of labels for all nestable objects
+
+Args:
+    filter_by_query_string: Whether the final query should accept a full text query string
+    filter_by_stable_target_id: Whether the final query should filter by stableTargetId
+
+Returns:
+    total: Count of all items found by this query
+    items: List of merged items, each item has the following attributes:
+        entityType: the type of the merged item
+        identifier: the identifier of the merged item
+        components: The rule and / or extracted items with the stableTargetId of this
```

Comment:
i thought we called those _fragments_ in some repo or documentation. but i couldn't find any search hits, so maybe it was just in my head 😄